AdobeStockImage/Model/Storage/Delete.php

```diff
@@ -52,7 +62,7 @@ public function execute(string $path): void
         try {
             $mediaDirectory = $this->filesystem->getDirectoryWrite(DirectoryList::MEDIA);
             if ($mediaDirectory->isFile($path)) {
```

Comment:
`isFile` is not necessary as it's performed by the `storage`. Probably we can remove this class completely and use the storage directly instead.

Reply:
ok